django/db/models/sql/compiler.py

```diff
@@ -1115,7 +1117,7 @@ def as_sql(self):
             raise EmptyResultSet
         sql, params = [], []
         for annotation in self.query.annotation_select.values():
-            agg_sql, agg_params = self.compile(annotation)
+            agg_sql, agg_params = self.compile(annotation, select_format=True)
```

Comment:
This is my first internal ORM patch of substance and I thought the name was fine, but I'm probably ignorant of why it's confusing (it seemed to match `SQLAggregateCompiler`).